secrets.yml.example

```diff
@@ -93,8 +93,27 @@ secrets:
         -----END RSA PRIVATE KEY-----
       public: >
         ssh-rsa a public ssh key goes here backup@backups
+    test_suite:
+      private: |
+        -----BEGIN RSA PRIVATE KEY-----
+        a private ssh key goes here
+        -----END RSA PRIVATE KEY-----
+      public: >
+        ssh-rsa a public ssh key goes here test-suite@test-suite
   letsencrypt:
     account_key: |
       -----BEGIN RSA PRIVATE KEY-----
       ThisIsNotARealKey
       -----END RSA PRIVATE KEY-----
+  bonnyci_test_suite:
```

Comment:
maybe, there's no reason this list needs to be private so it can also go in the bastion host_vars.